masonry/src/widget/button.rs

```diff
@@ -41,7 +41,7 @@ impl Button {
     ///
     /// let button = Button::new("Increment");
     /// ```
-    pub fn new(text: impl Into<ArcStr>) -> Button {
+    pub fn new(text: impl Into<ArcStr>) -> Self {
```

Comment:
Not a dealbreaker, but I think this PR has a few too many drive-by fixes given its already large size.

Reply:
Was fixed on main in the meantime, I don't think there's any other drive-by fixes though?